### PR TITLE
W5 2

### PR DESCRIPTION
**Pull request by cockochan**

### CODE REVIEW

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 1]
```diff
@@ -25,6 +35,14 @@ function exerciseOne(arrayOfPeople) {
  *
  */
 function exerciseTwo(shopping) {
+  var ul = document.createElement("ul");
+  content.appendChild(ul);
+  shopping.map(function(item){
+  let li = document.createElement("li");
+  ul.appendChild(li);
+  li.innerHTML = item;
+  })
```
Comment (ChrisOwen101) [inline week-5/Homework/mandatory/2-exercises/exercises.js:44]: Some small formatting errors here. Could you use something like this?

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: week-5/Homework/mandatory/2-exercises/exercises.js [hunk 2]
```diff
@@ -58,6 +76,47 @@ function exerciseTwo(shopping) {
...
+    let ul2 = document.createElement("ul");
+    p.appendChild(ul2);
+    books.map(function(book,index){
+      let li = document.createElement("li");
+      ul2.appendChild(li);
+      id = book.name
+      li.id =`${book.title}`;
+      li.innerHTML = (book.title +"  "+"♫"+"  "+ book.author)
+
+      let img = document.createElement("img");
```
Comment (ChrisOwen101) [inline week-5/Homework/mandatory/2-exercises/exercises.js:91]: There is quite a lot of code inside this function. When this happens it's often best to split it into small functions. Could you try this?

File: week-5/Homework/mandatory/3-project/js/main.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    nameInput.style.backgroundColor = ("white");
+    return true;
+}
+
+}
+
+let checkDescr = function(event){
+    let descrInput = document.getElementById("exampleTextarea");
+    console.log(descrInput.value);
+    if (descrInput.value.length <=0)
```
Comment (ChrisOwen101) [inline week-5/Homework/mandatory/3-project/js/main.js:80]: Can the length of the string be less than zero? 😉

File: week-5/Homework/mandatory/2-exercises/exercises.js (hunk 1 above)
Comment (cockochan) [reply week-5/Homework/mandatory/2-exercises/exercises.js:44]: Done. only one of my computers did not have prettier now fixed

File: week-5/Homework/mandatory/3-project/js/main.js (hunk 3 above)
Comment (cockochan) [reply week-5/Homework/mandatory/3-project/js/main.js:80]: Lol. That's a good finding

File: week-5/Homework/mandatory/3-project/js/main.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+orangeBtn.addEventListener("click", onOrangeBtn);
+
+
+let greenBtn =  document.getElementById("greenBtn");
+
+let onGreenBtn = function (){
+    let jumbo = document.getElementsByClassName("jumbotron");
+    jumbo[0].style.backgroundColor = "#87ca8a";
+    let donAb = document.getElementsByClassName("btn btn-primary btn-lrg");
+    donAb[0].style.backgroundColor = "black";
```
[the remark this answers is not in the document]
Comment (cockochan) [reply week-5/Homework/mandatory/3-project/js/main.js:35]: I hope soon to be confident with color numbers.

**Pull request closed without merging**